inc/namespace.php

```diff
@@ -69,6 +69,16 @@ function bootstrap() {
 
 	// Disable the admin_email verification interval.
 	add_filter( 'admin_email_check_interval', '__return_zero' );
+
+	// Performance enhancements.
+	// Comment pagination is always enabled.
```

Comment:
I'd further explain in this comment that "This is to prevent the CMS generating large HTML pages in cases where there may be 1000s of comments, as commenters will not see a cached version of the page"